receiver/googlecloudpubsubreceiver/README.md

```diff
@@ -12,18 +12,22 @@ This receiver gets OTLP messages from a Google Cloud [Pubsub](https://cloud.goog
 
 The following configuration options are supported:
 
+* `mode` (Optional): The mode that the receiver is in, either `push` or `pull` is supported. The default is `pull`.
```

Comment:
With this implementation, the receiver is either in `pull` or in `push` mode. Would it be possible to allow the receiver to be configured for both `pull` and `push` in a single instance? Does it make sense?

Reply:
You can set another receiver with another mode in the same collector... but if it would make sense? The choose of either `pull` or `push` depends on the deployment model. The `push` mode I'm going to use in Google Cloud Run (a serverless container runtime).